spec/factories/state_file_dependents.rb

```diff
@@ -0,0 +1,31 @@
+# == Schema Information
+#
+# Table name: state_file_dependents
+#
+#  id             :bigint           not null, primary key
+#  dob            :date
+#  first_name     :string
+#  intake_type    :string           not null
+#  last_name      :string
+#  middle_initial :string
+#  months_in_home :integer
+#  relationship   :string
+#  ssn            :string
+#  suffix         :string
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  intake_id      :bigint           not null
+#
+# Indexes
+#
+#  index_state_file_dependents_on_intake  (intake_type,intake_id)
+#
+FactoryBot.define do
```

Comment:
if the other DOB story from AZ get merged in before this we should do a rebase bc this is duplicated there